test-app/tests/integration/components/toucan-form/toucan-form-test.gts

```diff
@@ -278,15 +303,18 @@ module('Integration | Component | ToucanForm', function (hooks) {
     // Satisfy the validation and submit the form
     await fillIn('[data-textarea]', 'A comment.');
     await fillIn('[data-input]', 'CrowdStrike');
-    await click('[data-test-submit]');
     await click('[data-checkbox]');
     await click('[data-radio-2]');
     await click('[data-checkbox-group-2]');
 
+    await click('[data-test-submit]');
```

Comment:
This test worked previously due to the way revalidation happens on click with these elements.  This is where the submit _should_ be though.